EIPS/eip-7425.md

```diff
@@ -0,0 +1,175 @@
+
+## Motivation
+
+Tokenized reserves are an extension of tokenized vaults. The goal is to create a reserve similar to a real world reserve an entity has as a backup incase regular funds run low. In the real world, an entity will have certain criteria to access reserve funds. In a decentralized environment, an entity can incorporate stakeholders into their criteria. Hopefully this will help entities who participate in decentralized environments to be transparent with stakeholders.
+
+## Specification
+
+### Definitions:
+
+	- owner: The creator of the reserve.
+	- user: Stakeholders of specific proposals
+	- reserve: The tokenized reserve contract
+	- proposal: Occurs when the owner wants a withdrawal from contract
+ 
+### Constructor:
+ 
+ 	- name: ERC-20 token name
+  	- ticker: ERC-20 ticker
+   	- asset: ERC-4626 underlying ERC-20 address
+	- rAuth: Primary authorized user
+	- rOwner: Owner of the Reserve
+ 
+### Interface
+    
+``` solidity
+    interface TokenReserve{
+	    // @dev proposals event
+	    event proposals(
+	    	address indexed token,
+	    	uint256 indexed proposalNum,
+	    	uint256 indexed amount,
+	    	address recipient
+	    );
+	    /// @dev owner deposit event
+	    event depositR(
+	    	address indexed token,
+	    	uint256 indexed amount,
+	    	uint256 indexed time,
+	    	uint256 count
+	    );
+	    /** @dev Owner accounting struct
+	    * @param time Deposit time
+	    * @param token Address of ERC-20 token
+	    * @param deposit Amount of owner deposit
+	    */
+	    struct ownerAccount{
+	    	uint256 time;
+	    	address token;
+	    	uint256 deposit;
+	    }
+	    /** @dev User accounting struct
+	    * @param proposal Specific participating proposal number
+	    * @param deposit Amount user deposited
+	    * @param withdrew Amount of user withdrew
+	    */
+	    struct userAccount{
+	    	uint256 proposal;
+	    	uint256 deposit;
+	    	uint256 withdrew;
+	    }
+	    /** @dev Proposal accounting struct
+	    * @param token Address of ERC-20 token
+	    * @param withdrew Amount withdrawn
+	    * @param received Amount received
+	    */
+	    struct proposalAccount{
+	    	address token;
+	    	uint256 withdrew;
+	    	uint256 received;
+	    }
+	    /**
+	    * @dev Make a deposit to a proposal creating new shares
+	    * - MUST be opened proposal
+	    * - MUST NOT be opened proposal that was closed
+	    * NOTE: using the deposit() will cause shares to not be accounted for in a proposal
+	    * @param assets Amount being deposited
+	    * @param receiver Address of depositor
+	    * @param proposal Number associated proposal
+	    */
+	    function proposalDeposit(uint256 assets, address receiver, uint256 proposal) external virtual returns (uint256);
+	    /**
+	    * @dev Make a deposit to a proposal creating new shares
```

Comment:
This has the same comment as the previous function?

Reply:
I removed this functions to allow the implementor to decide if they want implement the mint() and redeem() of ERC-4626.